product_docs/docs/pgd/5/rel_notes/pgd_5.2.0_rel_notes.mdx

```diff
@@ -0,0 +1,54 @@
+| BDR       | 5.2.0   | Feature         | Allow a logical join of node if there are foreign key constraints violations |
+| BDR       | 5.2.0   | Enhancement     | Introduce receiver side transaction state tracking |
+| BDR       | 5.2.0   | Bug             | Change to `group_raft_details` view to avoid deadlock possibility |
+| BDR       | 5.2.0   | Bug             | Log the extension upgrade |
+| BDR       | 5.2.0   | Bug             | Add check for conflicting node names |
+| BDR       | 5.2.0   | Bug             | Fix a crash during Raft manual snapshot restore |
+| BDR       | 5.2.0   | Bug             | Don't try to establish consensus connections to parting or parted nodes |
+| BDR       | 5.2.0   | Bug             | Fix `tcp_user_timeout` GUC to use the correct unit |
+| BDR       | 5.2.0   | Bug             | Fix the consensus snapshot compatibility with PGD 3.7 |
+| BDR       | 5.2.0   | Bug             | Modify `bdr.stat_subscription` for new columns |
+| BDR       | 5.2.0   | Bug             | Address a crash when BDR is used together with pgaudit |
+| BDR       | 5.2.0   | Bug             | Skip parting synchronization to witness node |
+| BDR       | 5.2.0   | Bug             | Generate correct keepalive parameters in connection strings |
+| BDR       | 5.2.0   | Bug             | Enable various scenarios of switching nodes between groups and their subgroups. For example, transition node from a group to any of the nested sub-groups|
+| BDR       | 5.2.0   | Bug             | Reduce the amount of WAL produced by consensus on idle server |
+| BDR       | 5.2.0   | Bug             | Fixed deadlock on autopartition catalogs when a concurrent `DROP EXTENSION` is executing |
+| BDR       | 5.2.0   | Bug             | Fix sporadic failure when dropping extension after node restart |
+| BDR       | 5.2.0   | Bug             | Add workaround for crash due to pgaudit bug 212 |
+| BDR       | 5.2.0   | Bug             | Fix deadlock between consensus and global monitoring queries |
+| BDR       | 5.2.0   | Bug             | Fix query cancellation propagatation across `bdr.run_on_all_nodes` |
+| BDR       | 5.2.0   | Bug             | Disallow invoking `bdr.run_on_nodes()`, `bdr.run_on_group()` and `bdr.run_on_all_nodes()` on parted nodes |
+| CLI       | 5.2.0   | Feature         | Add new GUCs verification in `verify-settings` command |
+| CLI       | 5.2.0   | Bug fix         | Truncate the long value of GUC in tabular output of `verify-settings` |
+| CLI       | 5.2.0   | Bug fix         | Upgrade database driver library version which fixes `connect_timeout` issue when `sslmode=allow` or `sslmode=prefer` |
+| Proxy     | 5.2.0   | Feature         | Add HTTP(S) APIs for Proxy health check |
+| Proxy     | 5.2.0   | Enhancement     | Improve route change events handling mechanism |
+| Proxy     | 5.2.0   | Bug fix         | Upgrade database driver library version which fixes `connect_timeout` issue when `sslmode=allow` or `sslmode=prefer` |
```

Comment:
"Upgrade database library ..." is duplicated several lines above. Please remove one of them.

Reply:
One is for `CLI` and another for `proxy`.

Reply:
👍